[ZING-JVM] RingbufferAddReadOneStressTest.whenLongTTLAndSmallBuffer

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-1 completed error expected null, but was:<com.hazelcast.ringbuffer.StaleSequenceException: sequence:1472648 is too small. The current headSequence is:1472652 tailSequence is:1473651>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
	at com.hazelcast.ringbuffer.impl.RingbufferAddReadOneStressTest.test(RingbufferAddReadOneStressTest.java:82)
	at com.hazelcast.ringbuffer.impl.RingbufferAddReadOneStressTest.whenLongTTLAndSmallBuffer(RingbufferAddReadOneStressTest.java:47)
```

https://hazelcast-l337.ci.cloudbees.com/view/Zing/job/Hazelcast-3.x-ZingJDK8/25/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAddReadOneStressTest/whenLongTTLAndSmallBuffer/

```
java.lang.AssertionError: No error should have been thrown, but ConsumeThread-1 completed error expected null, but was:<com.hazelcast.ringbuffer.StaleSequenceException: sequence:1203808 is too small. The current headSequence is:1203812 tailSequence is:1204811>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotNull(Assert.java:755)
	at org.junit.Assert.assertNull(Assert.java:737)
	at com.hazelcast.test.TestThread.assertSucceedsEventually(TestThread.java:62)
	at com.hazelcast.ringbuffer.impl.RingbufferAddReadOneStressTest.test(RingbufferAddReadOneStressTest.java:82)
	at com.hazelcast.ringbuffer.impl.RingbufferAddReadOneStressTest.whenLongTTLAndSmallBuffer(RingbufferAddReadOneStressTest.java:47)
```

https://hazelcast-l337.ci.cloudbees.com/view/Zing/job/Hazelcast-3.x-ZingJDK7/38/testReport/junit/com.hazelcast.ringbuffer.impl/RingbufferAddReadOneStressTest/whenLongTTLAndSmallBuffer/

--- a/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAddReadOneStressTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/ringbuffer/impl/RingbufferAddReadOneStressTest.java
@@ -42,7 +42,7 @@ public class RingbufferAddReadOneStressTest extends HazelcastTestSupport {
     @Test
     public void whenLongTTLAndSmallBuffer() throws Exception {
         RingbufferConfig ringbufferConfig = new RingbufferConfig("foo")
-                .setCapacity(1000)
+                .setCapacity(3 * 1000)
                 .setTimeToLiveSeconds(30);
         test(ringbufferConfig);
     }

